.gitignore

```diff
@@ -0,0 +1,4 @@
+logs/
+target/
+RUNNING_PID
```

Comment:
any chance we can put this in a local `tmp` dir instead of the root?

Reply:
Do you mean the whole gitignore?  gitignore is shared / global.
https://git-scm.com/docs/gitignore

Reply:
I wasn't thinking straight with this comment, feel free to ignore